Scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,439 @@
+@prefix scenario: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/scenario/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+scenario:scenarioShape a sh:NodeShape ;
```

Comment:
Shapenames should be in PascalCase
`scenario:ScenarioShape`

Please apply this to the other shapes as well if applicable

Reply:
unresolve: it is still scenarioShape and not ScenarioShape

Reply:
fixed